mod/dfrn_notify.php

```diff
@@ -316,18 +316,22 @@ function dfrn_notify_content(App $a) {
 		$rino = get_config('system','rino_encrypt');
 		$rino = intval($rino);
 		// use RINO1 if mcrypt isn't installed and RINO2 was selected
-		if ($rino==2 and !function_exists('mcrypt_create_iv')) $rino=1;
+		/// @TODO Define a code-standard: and/AND/&& are around
```

Comment:
Going to fix this tomorrow. Time to turn myself in. !tzaf (time-zone appropriate farewell)